src/Raygun4php/RaygunClient.php

```diff
@@ -304,6 +342,61 @@ private function is_assoc($array)
       return (bool)count(array_filter(array_keys($array), 'is_string'));
     }
 
+    /**
+     * flushBundle
+     *
+     * Flush any leftover messages stored in the global session which weren't bundled
+     *
+     * @return [void]
+     */
+
+    private function flushBundle() {
+      $bundle = $this->bundler->getFromStorage();
+
+      if(!empty($bundle)) {
+        $this->bundler->setBundle($bundle);
+
+        if(count($bundle) === 1) {
+          $this->Send($bundle[0]);
```

Comment:
If we are setting the bundle above, then won't sending the error at this point mean duplicate errors being sent to the API? One with the `setBundle` and then again when we call the `Send` method here?